### ISSUE TITLE
Yarn audit failure

### ISSUE THREAD

**Issue by Introvertuous** [Shopify/polaris-react#1617]
## Issue summary

yarn audit

```
555 vulnerabilities found - Packages audited: 82316
Severity: 305 Low | 234 Moderate | 16 High
```

## Expected behavior

no vulnerabilities

## Actual behavior

```
555 vulnerabilities found - Packages audited: 82316
Severity: 305 Low | 234 Moderate | 16 High
```

Usually yarn audit failures are easy to fix by resetting yarn.lock, and upgrading large devDependencies like babel or jest.   It's not so straightforward here because many of the dependencies come in through @shopify/js-uploader and @shopify/sewing-kit.

## Steps to reproduce the problem

1.  git clone git@github.com:Shopify/polaris-react.git
1.  cd polaris-react
1.  yarn audit

## Specifications

- Polaris version number: 6e654ccf29d09406b1d1fda846ce4e8e46a67c02

**Comment by chloerice** [comment]
Thank you for raising this, @Introvertuous. Triaged and will discuss next steps with the team tomorrow.

**Comment by chloerice** [comment]
Hey @Introvertuous, just quick update--we're working on upgrades now. Most of the audit is noise, as its all development dependencies so there’s no risk to end users.

**Comment by Introvertuous** [comment]
Thank you for the quick response / action. It is true that a lot of the audit is caused via dev dependencies, but I think these should be resolved at some point (possibly as lower priority) because it is still a concern for anyone building polaris, which is what we are doing.

**Comment by BPScott** [comment]
We've fixed the majority of these warnings in #1624 and #1633 Once those two are merged we're down to "Severity: 6 Low | 8 Moderate". and most of the remaining issues are down to js-uploader using hardcoded old versions.
